src/proc/y12i-to-y16y16-mipi.cpp

```diff
@@ -16,7 +16,7 @@ namespace librealsense
     {
         auto count = width * height;
 #ifdef RS2_USE_CUDA
-        rscuda::split_frame_y16_y16_from_y12i_cuda_mipi(dest, count, reinterpret_cast<const y12i_pixel_mipi *>(source));
+        rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const rscuda::y12i_pixel_mipi *>(source));
```

Comment:
Instead of :rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const rscuda::y12i_pixel_mipi *>(source));
Please try:
rscuda::split_frame_y16_y16_from_y12i_cuda<rscuda::y12i_pixel_mipi *>(dest, count, reinterpret_cast<const rscuda::y12i_pixel_mipi *>(source));